cmd/main.go

```diff
@@ -110,10 +110,21 @@ func main() {
 	nodeName := os.Getenv("NODE_NAME")
 	node, err := k8sClientset.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})
 	if err != nil {
-		logger.Info("Error retrieving the Node from the index for a given node. Error :", zap.Error(err))
 		if errors.IsNotFound(err) {
```

Comment:
please keep this in the `ErrorRetry` method

Reply:
Done